tasks/google/add-amazon-linux-2/task.yaml

```diff
@@ -0,0 +1,111 @@
+summary: Add new debian sid image to google account
+
+environment:
+    BUCKET_NAME: spread-images
+    IMAGE_FILE: amazon-linux-2-64
+    IMAGE_URL: https://cdn.amazonlinux.com/os-images/2017.12.0.20180222/kvm/amzn2-kvm-2017.12.0.20180222-x86_64.xfs.gpt.qcow2
+    IMAGE: amazon-linux-2-64-v20180406
+    FAMILY: amazon-linux-2-64
+    DESCRIPTION: Amazon Linux 2 64 bits
+    DISK_NAME: disk.raw     
+
+execute: |
+    . "$TESTSLIB/google.sh"
+
+    if ! [[ "$SPREAD_SYSTEM" == ubuntu-* ]]; then
+        echo "System used to create amazon linux has to be ubuntu"
+        exit 1
+    fi
+
+    # Get the raw image
+    wget -O "${IMAGE_FILE}.qcow2" "$IMAGE_URL"
+    qemu-img convert "${IMAGE_FILE}.qcow2" "$DISK_NAME"
```

Comment:
IMO it's better to specify the output format explicitly: `qemu-img convert -O raw "${IMAGE_FILE}.qcow2" "$DISK_NAME"`